index.js

```diff
@@ -286,8 +285,18 @@ class Raspi extends EventEmitter {
   }
 
   pinMode(pin, mode) {
+    this._pinMode({ pin, mode });
+  }
+
+  _pinMode(options) {
+    const { pin, mode } = options;
```

Comment:
Another style nit: let's do the destructuring in the method signature, e.g.

```
[pinMode]({ pin, mode, pullResistor }) {
```

and drop this line here. I think it will be cleaner that way